src/libLookup/Lookup.cpp

```diff
@@ -2142,7 +2142,8 @@ bool Lookup::ProcessSetStateDeltasFromSeed(const bytes& message,
         return false;
       }
       BlockStorage::GetBlockStorage().PutStateDelta(txBlkNum, delta);
-      if (txBlkNum + 1 % NUM_FINAL_BLOCK_PER_POW == 0) {
+      if (((txBlkNum + 1) % NUM_FINAL_BLOCK_PER_POW == 0) &&
+          (txBlkNum + NUM_FINAL_BLOCK_PER_POW < highBlockNum)) {
         if (!AccountStore::GetInstance().MoveUpdatesToDisk()) {
```

Comment:
No param passing to MoveUpdatesToDisk means don't do repopulate. Aren't you going to repopulate at this point?

Reply:
You are correct. This change is already made in PR#1540.